pages/guides/dwolla-js/index.mdx

```diff
@@ -1,9 +1,16 @@
 ---
+layout: guides
+title: "Overview"
 group:
   category: Funding Sources
-  title: Add Bank with Dwolla.js
-
-title: Overview
+  title: Dwolla.js
+guide:
+  step: 0
+  filter: get-building
+  icon: guides-icon.svg
+meta:
+  title: Dwolla.js - Securely Transmit Bank Data | Dwolla API Documentation
+  description: Dwolla.js is a client-side JavaScript library designed to quickly onboard customers, add and verify bank accounts (IAV), and enable on-demand bank transfers.
 ---
```

Comment:
This file appears to be a duplicate of `pages/guides/dwolla-js/add-a-bank-account/index.mdx`